src/test/kotlin/org/jetbrains/exposed/sql/tests/h2/H2Tests.kt

```diff
@@ -0,0 +1,52 @@
+package org.jetbrains.exposed.sql.tests.h2
+
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.replace
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.TestDB
+import org.junit.BeforeClass
+import org.junit.Test
+import kotlin.test.assertFailsWith
+
+class H2Tests : DatabaseTestsBase() {
+
+    companion object {
+        @BeforeClass
+        @JvmStatic
+        fun setup(){
```

Comment:
I don't like approach on setting system properties in tests. If that test will be executed in first place, then all other tests will be run only on h2/h2_mysql. I guess it's ok to just remove this section because we run tests on all DB at TeamCity and withDb(TestDB.H2) is ok to exclude other DB from a concrete test.

Reply:
Done

Reply:
@Tapac I have found the DUPLICATE COLUMN problem.
When you use H2 in standard mode, it stores tables names in lowercase, but when you use MySQL mode is does in upperCase. 
```Default.kt``` has a lazy val with that information, so when i try to use 2 connections to H2, both standar and MySQL, the second fails to retrieve table's columns and tries to creates them causing DUPLICATE error.
Tell me if it make sense to you to fix this case, or if you are ok with excluding H2 in MySQL mode from those failing tests as i commited.